hibernate-core/src/main/java/org/hibernate/metamodel/internal/MetadataContext.java

```diff
@@ -636,7 +636,7 @@ private <X> void registerAttribute(Class<?> metamodelClass, Attribute<X, ?> attr
 				// mismatch in how the annotation processor interpreted the attribute and how our metamodel
 				// and/or annotation binder did.
 
-//              This is particularly the case as arrays are nto handled propery by the StaticMetamodel generator
+//              This is particularly the case as arrays are nto handled property by the StaticMetamodel generator
 
```

Comment:
I would say that nto should become 'not', and i think that 'propery' was meant to be 'properly'

Reply:
thanks. I updated.